Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
+    let age: BigUint = inp.trim().parse().expect("Not a positive integer.");
+    age
+}
+
+// turn age in years to days
+fn calc_days(age: BigUint) -> BigUint {
+    let days_in_a_year: BigUint = 365_i32
```

Comment:
I think you can just [multiply a u32 with a biguint](https://docs.rs/num-bigint/latest/num_bigint/struct.BigUint.html#impl-Mul%3C%26u32%3E-for-%26BigUint)
Also, define a [constant](https://doc.rust-lang.org/rust-by-example/custom_types/constants.html) rather than a let binding